utilities/media.ts

```diff
@@ -10,29 +10,29 @@ const sizes = {
 };
 
 // iterate through the sizes and create a media template
-export const media: any = Object.keys(sizes).reduce((accumulator, label) => {
+export const media = (Object.keys(sizes) as (keyof typeof sizes)[]).reduce((accumulator, label) => {
   // use em in breakpoints to work properly cross-browser and support users
   // changing their browsers font-size: https://zellwk.com/blog/media-query-units/
   const emSize = sizes[label] / 16;
 
   if (label === 'giant' || label === 'desktop') {
-    accumulator[label] = (...args) => css`
+    accumulator[label] = (first, ...interpolations) => css`
```

Comment:
Why the `first, ...interpolations` pattern?

Reply:
The `css` function from `styled-components` has the following signature:
```ts
css(first: TemplateStringsArray | CSSObject, ...interpolations: SimpleInterpolation[]): FlattenSimpleInterpolation
```
Therefore, in strict mode, Typescript isn't allowing to use a single rest parameter like this `css(...args)`